CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-## dbt-snowflake 1.0.0 (Release TBD)
+## dbt-snowflake next
+Resolves an issue caused when the Snowflake OCSP server is not accessible. This change only exposes the insecure_mode boolean avalable in the Snowflake python connector to dbt. This allows dbt to pass insecure_mode=true to the connector.
```

Comment:
You can leave the title as is, add your message to an "Under the hood" section, and add yourself as a contributor:
```suggestion
## dbt-snowflake 1.0.0 (Release TBD)

### Under the hood
Resolves an issue caused when the Snowflake OCSP server is not accessible. This change only exposes the `insecure_mode` boolean avalable in the Snowflake python connector to dbt. This allows dbt to pass `insecure_mode=true` to the connector. ([#31](https://github.com/dbt-labs/dbt-snowflake/issues/31), [#49](https://github.com/dbt-labs/dbt-snowflake/pull/49))

### Contributros
```